src/allmydata/test/no_network.py

```diff
@@ -59,14 +62,24 @@ class IntentionalError(Exception):
 class Marker(object):
     pass
 
+fireNow = partial(defer.succeed, None)
+
 class LocalWrapper(object):
```

Comment:
If you've already figured out what `LocalWrapper` is/does adding a docstring would be great